.devcontainer/devcontainer.json

```diff
@@ -7,23 +7,32 @@
   "containerEnv": { "DEVCONTAINER": "1" },
   "appPort": 8123,
   "runArgs": ["-e", "GIT_EDITOR=code --wait"],
+  // Please keep this file in sync with settings in home-assistant/.vscode/extensions.json
```

Comment:
These have different goals. It is what is installed vs recommendations. This comment is invalid IMHO for that reason.